web-common/templates/Component/Stateful.purs.template

```diff
@@ -0,0 +1,68 @@
+-- | This module contains the implementation and entry point of the component
```

Comment:
as I mentioned in the call, I rather continue having a new component divided into the different files, and eventually have an Index Module

Reply:
For private types, what do you think about having `Component.$name.Internal.Types`, `Component.$name.Internal.State`, and `Component.$name.Internal.View` then, just to drive home that these are not part of the public API? Or even just having `Component.$name.Types.Internal` so that the public types and the private types don't cross streams?

Reply:
We could do that taking into account that we use a generator.